mmcls/datasets/builder.py

```diff
@@ -23,6 +23,25 @@
 SAMPLERS = Registry('sampler')
 
 
+def get_sampler_cfg(cfg, distributed):
```

Comment:
MME need to handle `DistributedSampler` and `InfiniteSampler` as default sampler when runner type is different, so there is a `set_default_sampler_cfg` function. 
But MMClassification only has one default sampler: `DistributedSampler`. I think this function is unnecessary here.